supertokens/models.go

```diff
@@ -47,6 +47,8 @@ type TypeInput struct {
 	RecipeList            []Recipe
 	Telemetry             *bool
 	OnSuperTokensAPIError func(err error, req *http.Request, res http.ResponseWriter)
+	Log                   Logger
+	RequestIDKey          any
```

Comment:
We should avoid using any

Reply:
The [context](https://cs.opensource.google/go/go/+/refs/tags/go1.20.6:src/context/context.go;l=159) accepts `any` as the key and value type, so specifying a type here may not work.

I also noticed the [chi middleware](https://github.com/go-chi/chi/blob/99bc99e0bf0edc147e82765f2848167922a5c644/middleware/request_id.go#L18) for request ID uses `int` instead of a string type.